LibrariesPrototype/Common/Helpers/Session.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Session.swift
+//  LibrariesPrototype
+//
+//  Created by Ozgun Emre Zor on 23/04/2019.
+//  Copyright © 2019 Ozgun Zor. All rights reserved.
+//
+
+import Foundation
+
+class Session {
```

Comment:
just a comment, private init looks nice on singleton.

Reply:
like what? Can you please write some here?

Reply:
just adding
```suggestion
private init() {

}
```
if your class can only be used on ' shared ' variable.